plugins/modules/clickhouse_info.py

```diff
@@ -532,6 +539,47 @@ def get_quotas(module, client):
     return quota_info
 
 
+def get_all_grants(module, client):
+    """Get grants.
+
+    Returns a dictionary with users and roles names as keys.
+    """
+    query = ("SELECT user_name, role_name, access_type, database, "
+             "table, column, is_partial_revoke, grant_option FROM system.grants")
+
+    result = execute_query(module, client, query)
+
+    if result == PRIV_ERR_CODE:
+        return {PRIV_ERR_CODE: "Not enough privileges"}
+
+    grants_info = {'user_name': {},
+                   'role_name': {},
+                   }
```

Comment:
```suggestion
    grants_info = {
        'user_name': {},
        'role_name': {},
    }
```
How about using `users` and `groups` instead ?
If I understand it correctly those dicts can contain multiple users and groups, right?
It'd look like:
```
grants_info: {
    users: {
        bob: { ... },
        alice: { ... },
    },
   roles: {
       accountant: { ... },
       sales: { ... },
   },
}
```